src/read/elf/file.rs

```diff
@@ -173,6 +173,24 @@ where
     ) -> Option<ElfSection<'data, 'file, Elf, R>> {
         None
     }
+
+    fn elf_verneed_by_version(&self, version: VersionIndex) -> read::Result<Option<(&'data elf::Verneed<Elf::Endian>, SectionIndex)>> {
+        let svt = self.sections.versions(self.endian, self.data)?;
```

Comment:
We don't want to call this for every import. We've already called it in `imports`, so `svt` could be passed as a parameter to this function.